configure.ac

```diff
@@ -167,6 +167,13 @@ AC_ARG_ENABLE([zmq],
 
 AC_ARG_WITH([protoc-bindir],[AS_HELP_STRING([--with-protoc-bindir=BIN_DIR],[specify protoc bin path])], [protoc_bin_path=$withval], [])
 
+# Enable manpages
+AC_ARG_ENABLE(man,
+    [AS_HELP_STRING([--enable-man],
```

Comment:
Makes sense after looking at the other configure options. Would this be better:
```
+    [AS_HELP_STRING([--disable-man],
+                    [do not install man pages (default is to install)])],,
```
And should I remove "# Enable manpages" comment? only --disable-wallet and --debug has a comment everything else doesn't.